astroquery/esa/hubble/core.py

```diff
@@ -119,14 +119,79 @@ def download_product(self, observation_id, *, calibration_level=None,
 
         shutil.move(response, filename)
 
+    def get_member_observations(self, observation_id):
+        """
+        Parameters
+        ----------
+        observation_id : str, mandatory
+            Observation identifier.
+
+        Returns
+        -------
+        A list of strings with the observation_id of the associated
+        observations that can be used in get_product_list and
+        get_obs_products functions
+        """
+        if observation_id is None:
+            raise ValueError(self.REQUESTED_OBSERVATION_ID)
+        observation_type = self.get_observation_type(observation_id)
+
+        if 'Composite' in observation_type:
+            oids = self._select_members(observation_id)
+        elif 'Simple' in observation_type:
+            oids = self._select_composite(observation_id)
+        else:
+            raise ValueError("Invalid observation id")
+        return oids
+
+    def get_hap_hst_link(self, observation_id):
```

Comment:
since this is public, could you add a docstring?

Reply:
The docsting has been added